### DISCUSSION TITLE
RoadMap

### DISCUSSION THREAD

**Discussion by sendya** [vueComponent/pro-layout#172] [General]
- 2x
  - [x] add `menu` custom selectedKeys | openKeys | defaultOpenKeys
  - [x] add `watermark` component
  - [ ] update `multi-tabs`
- 3.1.x (2021)
  - provide / inject optimization
  - pro-layout props optimization
- 3.2.x (2022)
  - [x] support Vue3.2 and Ant Design Vue@3x(alpha|beta|release) 
  - [ ] custom render optimization
  - [ ] menuData / breadcrumbData generator util
  - [x] support JSX
  - [ ] add more docs

**Comment by YogaZong** [comment]
Support.

**Comment by watsonhaw5566** [comment]
multi-tabs 是个大学问

**Comment by SnowGuest** [comment]
期待3.2版本，3.2才是pro-layout最棒的版本

**Comment by Sgenmi** [comment]
项目名子，都改了，看到 advue 3.x了
